Libraries/Boards/MAX32690/FTHR_APPS_P1/Include/board.h

```diff
@@ -0,0 +1,124 @@
+/******************************************************************************
+ * Copyright (C) 2023 Maxim Integrated Products, Inc., All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL MAXIM INTEGRATED BE LIABLE FOR ANY CLAIM, DAMAGES
+ * OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * Except as contained in this notice, the name of Maxim Integrated
+ * Products, Inc. shall not be used except as stated in the Maxim Integrated
+ * Products, Inc. Branding Policy.
+ *
+ * The mere transfer of this software does not imply any licenses
+ * of trade secrets, proprietary technology, copyrights, patents,
+ * trademarks, maskwork rights, or any other form of intellectual
+ * property whatsoever. Maxim Integrated Products, Inc. retains all
+ * ownership rights.
+ *
+ ******************************************************************************/
+
+/**
+ * @file    board.h
+ * @brief   Board support package API.
+ */
+
+#include <stdio.h>
+
+#ifndef LIBRARIES_BOARDS_MAX32690_FTHR_APPS_P1_INCLUDE_BOARD_H_
+#define LIBRARIES_BOARDS_MAX32690_FTHR_APPS_P1_INCLUDE_BOARD_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef CONSOLE_UART
+#define CONSOLE_UART 0 /// UART instance to use for console
+#endif
+
+#ifndef CONSOLE_BAUD
+#define CONSOLE_BAUD 115200 /// Console baud rate
+#endif
+
+#ifndef HCI_UART
+#define HCI_UART 3
+#endif
+
+#ifndef TERMINAL_UART
+#define TERMINAL_UART CONSOLE_UART
+#endif
+
+#ifndef USER_UART
+#define USER_UART 1
+#endif
+
+#ifdef LED_OFF
+#undef LED_OFF
+#endif
+#define LED_OFF 1 /// Override inactive state of LEDs
+
+#ifdef LED_ON
+#undef LED_ON
+#endif
+#define LED_ON 0 /// Override active state of LEDs
+
+#ifndef EXT_FLASH_BAUD
+#define EXT_FLASH_BAUD 4000000
+#endif
+
+#define FTHR_APPS_P1 1 /// Used in examples to control program flow.
```

Comment:
Lets double check board name with hw team, 
they may just wish to call the board as FTHR.

Reply:
HW team prefer just FTHR, please rename it as FTHR to simplify things.

Reply:
Still see `FTHR_APPS_P1` defined above, should this have been updated?